worker/transcribee_worker/identify_speakers.py

```diff
@@ -7,73 +7,80 @@
+        if classifier is None:
+            raise ValueError("classifier is None")
+
+        embeddings = []
+        for i, (start, end) in enumerate(segments):
+            progress_callback("generating speaker embeddings", i / (len(segments) + 1))
+            wav = audio[start:end]
+            wav_tensor = torch.tensor(wav[np.newaxis, :])
+            embedding = classifier.encode_batch(wav_tensor)
+            embeddings.append(embedding[0, 0].detach().numpy())
 
-    classifier = EncoderClassifier.from_hparams(
-        source="speechbrain/spkrec-ecapa-voxceleb",
-        savedir=".data/models/speechbrain-spkrec-ecapa-voxceleb",
-    )
-    if classifier is None:
-        raise ValueError("classifier is None")
+        progress_callback(
+            "clustering speaker embeddings", len(segments) / (len(segments) + 1)
+        )
+        clustering = AgglomerativeClustering(
+            compute_full_tree=True,
+            linkage="complete",
+            n_clusters=None,
+            # distance_threshold curtesty of
+            # https://huggingface.co/pyannote/speaker-diarization/blob/369ac1852c71759894a48c9bb1c6f499a54862fe/config.yaml#L15
+            distance_threshold=0.7153,
+            metric="cosine",
+        )
+        clustering.fit(np.array(embeddings))
 
-    embeddings = []
-    for i, (start, end) in enumerate(segments):
-        progress_callback("generating speaker embeddings", i / (len(segments) + 1))
-        wav = audio[start:end]
-        wav_tensor = torch.tensor(wav[np.newaxis, :])
-        embedding = classifier.encode_batch(wav_tensor)
-        embeddings.append(embedding[0, 0].detach().numpy())
+        # we now re-shuffle the labels so that the first occuring speaker is 0, the second is 1, ...
+        label_map = {}
+        for label in clustering.labels_:
+            if label not in label_map:
+                label_map[label] = str(len(label_map))
 
-    progress_callback(
-        "clustering speaker embeddings", len(segments) / (len(segments) + 1)
-    )
-    clustering = AgglomerativeClustering(
-        compute_full_tree=True,
-        linkage="complete",
-        n_clusters=None,
-        # distance_threshold curtesty of
-        # https://huggingface.co/pyannote/speaker-diarization/blob/369ac1852c71759894a48c9bb1c6f499a54862fe/config.yaml#L15
-        distance_threshold=0.7153,
-        metric="cosine",
-    )
-    clustering.fit(np.array(embeddings))
+        for para, label in zip(doc.children, clustering.labels_):
+            para.speaker = label_map[label]
 
-    # we now re-shuffle the labels so that the first occuring speaker is 0, the second is 1, ...
-    label_map = {}
-    for label in clustering.labels_:
-        if label not in label_map:
-            label_map[label] = str(len(label_map))
+        queue.submit(None)
 
-    for para, label in zip(doc.children, clustering.labels_):
-        para.speaker = label_map[label]
+    return await anext(aiter(async_task(work)))
```

Comment:
`await anext(aiter(async_task(work)))` returns None, right?

```suggestion
   await anext(aiter(async_task(work)))
```